fronts-client/src/components/card/CardContent.tsx

```diff
@@ -1,9 +1,11 @@
 import styled from 'styled-components';
 import { CardSizes } from 'types/Collection';
+import { theme } from '../../constants/theme';
 
 const CardContent = styled.div<{
   displaySize?: CardSizes;
   textSize?: CardSizes;
+  isContentNotFound?: boolean;
```

Comment:
I would suggest to specify the "warning" state as a seperate CSS class (something like

```typescript
const NotFoundWarningStyle = css`
  background-color: theme.colors.greyMediumLighter;
...
 `

```

and then link that into the `CardContent` div below using the `css` prop, i.e.

```tsx
<CardContent css={isContentNotFound ? NotFoundWarningStyle : undefined}>

```

That makes it much easier to evaluate in test classes, because you can inspect the class-names on the div from within your test